### ISSUE TITLE
Device configuration lost if device is not connected

### ISSUE THREAD

**Issue by VitorAbyssal** [JaydenMaalouf/JoystickPlugin#49]
If I disconnect the device I configured, the Events do not appear on Editor Restart.

With Connected Device and Configuration set:

![image](https://github.com/JaydenMaalouf/JoystickPlugin/assets/82595178/706aa7fa-2cf2-4e57-8cfb-35c29f5e1537)

But if I disconnect the device, this happens:

![image](https://github.com/JaydenMaalouf/JoystickPlugin/assets/82595178/57bba1ac-af18-4fee-83a2-84a84291ca12)

With warning being: 
"InputAxisKey Event specifies invalid FKey'Joystick_TFlightHotasX_0_Axis0' for  Joystick_TFlightHotasX_0_Axis0"

And if I search any input Event, they do not come up

![image](https://github.com/JaydenMaalouf/JoystickPlugin/assets/82595178/977fc18e-6642-4b83-a1dd-f160b3edf461).

It seems the events depend on whether there's a configuration AND if the device is connected, but we don't want to be dependant on the device being connected or not. Plus, we have an array of Joysticks to use and configure and we can't have all of them connected at the same time.

I tried to replicate with two joysticks at the same time and the same happens: while both configured and connected, they work, but if I disconnect one of them, its events disappear

**Comment by VitorAbyssal** [comment]
An update: turns out if I compile with the device connected, the event shows up again, which won't be a problem with Inputting on Input Actions. But for some reason I cannot seem to map any axis, only buttons

**Comment by VitorAbyssal** [comment]
Okay, second and hopeful final update: 

On my project, we have two possible pawns to control through two different player controllers and on the same device. In order to handle this, we had to override the Game Viewport Client and handle input to the correct pawn there using an overridable InputKey function.

What I noticed here with the plugin was that the Keys from the Joystick were being recognized by the First Pawn as well as the Second Pawn, but the Axis were only on the First pawn. This made me realize I needed to override the InputAxis function on the Game Viewport Client similar to the InputKey function in order to get them working and alas they do!

So finally the project is handling the input from the joystick correctly :D
